Software/Development/SemesterProject/App/systemManager.c

```diff
@@ -27,7 +29,7 @@
 static SysMgrState SystemState = STARTUP;
 
 
-void systemManager(void)
+void manageSystem(void)
```

Comment:
Why change name? systemManager is better...

Reply:
Discussed, having a verb in the functions namesI think is better.